packages/stacked_generator/lib/src/generators/router/generator/arguments_class/argument_class_builder_helper.dart

```diff
@@ -26,6 +26,14 @@ class ArgumentClassBuilderHelper {
         .toList();
   }
 
+  List<String> get convertParametersToStrings {
```

Comment:
Can we add a duplicate of this to convert the parameters to a Map. I think that would be quite useful to print out in that way. Key = name, value  = parameterValue.